scripts/validate_unwanted_patterns.py

```diff
@@ -23,6 +23,7 @@
     "_interval_shared_docs",
     "_merge_doc",
     "_shared_docs",
+    "_shared_doc_kwargs",
```

Comment:
i think we by definition *don't* import this across modules, you are circumventing this. i believe this will impact many doc-strings in a way we don't want.

Reply:
Alright. I couldn't think of an unwanted side effect, so perhaps smarter to repeat the shared kwargs in two files instead of this.